fabbwled-backend/src/main/resources/sections/001-036.yaml

```diff
@@ -0,0 +1,10 @@
+- number: 1
+  content:
+    - text: |
+        The  approach  of  dawn  has  turned  the  sky  a  milky  grey-green,like jade. The sea is a luminous pane of silver. Holding the tiller of  your  sailing  boat,  you  keep  your  gaze  fixed  on  the  glittering constellation  known  as  the  Spider.  
```

Comment:
many double-spaces in here :)

Reply:
also, please don't have lines that are this long, we should limit ourselves to ~100 characters